Tasks/NugetPublisher/task.json

```diff
@@ -0,0 +1,66 @@
+{
+    "id": "333b11bd-d341-40d9-afcf-b32d5ce6f25b",
+    "name": "NuGetPublisher",
+    "friendlyName": "NuGet Publisher",
+    "description": "Uploads nupkg files to a nuget server",
+    "helpMarkDown": " ",
+    "category": "Package",
+    "author": "Lawrence Gripper",
+    "version": {
+        "Major": 0,
+        "Minor": 1,
+        "Patch": 36
+    },
+    "minimumAgentVersion": "1.83.0",
+    "groups": [
+        {
+            "name":"advanced",
+            "displayName":"Advanced",
+            "isExpanded":false
+        }
+    ],    
+    "instanceNameFormat": "NuGet Publisher $(solution)",
+    "inputs": [
+        {
+            "name": "searchPattern",
+            "type": "filePath",
+            "label": "Path/Pattern to nupkg",
+            "defaultValue":"**\\*.nupkg",
+            "helpMarkDown": "The pattern to match or path to nupkg files to be uploaded",
+            "required":true
+        },
+        {
+            "name": "connectedServiceName",
+            "type": "connectedService:Generic",
+            "label": "Nuget Server Endpoint",
+            "required": true,
+            "helpMarkDown": "The nuget server generic endpoint, set the key 'Password/Token Key' field to your nuget api key"
```

Comment:
IMHO, it could be missunderstod NuGet API key should be entered in the service endpoint field. Something like this might be clearer: 
> The nuget server generic endpoint. For the endpoint set the key 'Password/Token Key' field to your NuGet api key